packages/screens/LaunchpadERC20/LaunchpadERC20Sales/LaunchpadERC20CreateSaleFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { View } from "react-native";
+
+import FlexRow from "../../../components/FlexRow";
+import { PrimaryButton } from "../../../components/buttons/PrimaryButton";
+import { SecondaryButton } from "../../../components/buttons/SecondaryButton";
+import { Separator } from "../../../components/separators/Separator";
+import { layout } from "../../../utils/style/layout";
+import { useCreateSaleState } from "../hooks/useCreateSale";
+
+const DEFAULT_WIDTH = 120;
+
+export const LaunchpadERC20CreateSaleFooter: React.FC<{
+  onSubmit: () => void;
+  disableNext: boolean;
+  nextText?: string;
+  backText?: string;
+  width?: number;
+}> = (props) => {
+  const {
+    actions: { goPrevStep },
+  } = useCreateSaleState();
+
+  return (
+    <View>
+      <Separator
+        style={{
+          marginVertical: layout.spacing_x2,
+        }}
+      />
+
+      <FlexRow style={{ justifyContent: "space-between" }}>
+        <SecondaryButton
+          onPress={goPrevStep}
+          text="Back"
+          size="SM"
+          width={props?.width || DEFAULT_WIDTH}
+        />
+        <PrimaryButton
+          onPress={props.onSubmit}
+          text={props.nextText || "Next"}
+          size="SM"
+          width={props?.width || DEFAULT_WIDTH}
```

Comment:
Please remove `width` usage from this `PrimaryButton` to let it grow depending on the text.
Before :  ![image](https://github.com/user-attachments/assets/2d2019bd-7559-450e-8e4d-36cc8a5a15ad) After : ![image](https://github.com/user-attachments/assets/65693a7e-f2c9-47fa-bd40-f313290a8e1f)

Reply:
[e38dcba](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/e38dcbadc09e61bedb5411ad7bae60e20749f4d6)